docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
+
+We support linking to the source code on GitHub, or any other VCS provider.
+
+![Link to GitHub](../../images/guide/link-to-github.gif)
+
+## Usage
+
+To enable source code linking with Logfire, you need to configure your project. Here's how:
+
+```python
+import logfire
+
+logfire.configure(
+    code_source=logfire.CodeSource(
+        repository='https://github.com/pydantic/logfire',
+        revision='main',
+        root_path='.',
+    )
+)
+```
+
+You can check more about it on the [`logfire.CodeSource`][logfire.CodeSource] on our API reference.
+
+| Parameter | Description |
```

Comment:
This table feels redundant with the example. What about adding comments to the example? Maybe those clickable bubbles?

Reply:
yes, bubbles.